server-common/src/main/java/org/apache/kafka/server/common/MetadataVersion.java

```diff
@@ -232,7 +233,7 @@ public enum MetadataVersion {
      * <strong>Think carefully before you update this value. ONCE A METADATA VERSION IS PRODUCTION,
      * IT CANNOT BE CHANGED.</strong>
      */
-    public static final MetadataVersion LATEST_PRODUCTION = IBP_3_7_IV4;
+    public static final MetadataVersion LATEST_PRODUCTION = IBP_3_8_IV0;
```

Comment:
I guess this single line needs to be backported to 3.8.

Reply:
The whole PR needs to be backported to 3.8.

But yes, this line as well :)